style.css

```diff
@@ -160,3 +160,43 @@ figcaption {
   font-size: 24px;
   font-weight: bold;
 }
+
+/*About us section*/
+
+#about-us {
+  width: 1240px;
+  height: 679px;
+  margin-left: 116px;
+  margin-right: 84px;
+}
+
+#about-us h2 {
+  color: #293241;
+  margin: 0 50px;
+}
+
+.about-us-container {
+  display: flex;
+  flex-direction: row;
+}
+.about-us-text {
+  margin-top: 30px;
+  font-size: 24px;
+}
+.about-us-text-1 {
+  width: 628px;
+}
+.about-us-text-2 {
+  width: 664px;
+}
+.about-us-text-3 {
+  width: 579px;
+}
+span {
```

Comment:
šis elementas html faile turi priskirtą klasę ir ją geriausiai ir naudoti. apskritai nėra labai gerai naudoti pačius elementus, kaip selectorius, nes tuomet šis stilius galios visiems `span` elementams. Taip gali atsirasti netikėtų klaidų. Mažuose projektuose kaip šis, lengva viską atsiminti ir susekti, nėra daug pasikartojančių elementų, bet kodo kiekiui cugant, lengva nepastebėti. Todėl visada geriausia priskirti klasę, norint pridėti stilių.